src/components/pageData.js

```diff
@@ -26,7 +25,7 @@ const PageData = class PageData {
       }
     }
     for (let i = 0; i < data.length; i++) {
-      let newFile = extraFiles ? { } : clone(this.params.files[ fileName ])
+      let newFile = extraFiles ? { } : JSON.parse(JSON.stringify(this.params.files[ fileName ]))
```

Comment:
👍

Reply:
Yes, though I would assume there is a low chance of functions being passed here.